.ci/Jenkinsfile

```
@@ -19,14 +21,15 @@ pipeline {
     disableResume()
     durabilityHint('PERFORMANCE_OPTIMIZED')
     quietPeriod(10)
-    rateLimitBuilds(throttle: [count: 60, durationName: 'hour', userBoost: true])
+    rateLimitBuilds(throttle: [count: 4, durationName: 'hour', userBoost: true])
   }
   triggers {
     issueCommentTrigger('(?i).*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tests(?:\\W+please)?.*')
+    upstream('Beats/beats-beats-mbp/master')
```

Use the current MBP:
- https://beats-ci.elastic.co/job/Beats/job/beats/job/master/

```suggestion
    upstream('Beats/beats/master')
```